docs/en/remote-flow.rst

```diff
@@ -141,17 +143,19 @@ response, containing the request URI, are described in the Table below.
   * - **Name**
     - **Description**
   * - **client_id**
-    - Unique identifier of the Relying Party.
+    - REQUIRED. Unique identifier of the Relying Party.
   * - **request_uri**
-    - The HTTPs URL where the Relying Party provides the signed Request Object to the Wallet Instance. 
-    
+    - REQUIRED. The HTTPs URL where the Relying Party provides the signed Request Object to the Wallet Instance.
+  * - **client_id_scheme**
+    - OPTIONAL. The scheme used by the Relying Party for the client_id, detailing the format and structure and the trust evaluation method. It SHOULD be set with ``entity_id``.
+  * - **request_uri_method**
+    - OPTIONAL. The HTTP method MUST be set with ``get`` or ``post``. The Wallet Instance should use this method obtain the signed Request Object from the request_uri. If not provided, the Wallet Instance SHOULD use the HTTP method ``get``. If provided, the Wallet Instance SHOULD provide its metadata within the HTTP POST body encoded in ``application/json``.    
```

Comment:
```suggestion
    - OPTIONAL. The HTTP method MUST be set with ``get`` or ``post``. The Wallet Instance should use this method to obtain the signed Request Object from the request_uri. If not provided or equal to ``get``, the Wallet Instance SHOULD use the HTTP method ``get``. Otherwise, the Wallet Instance SHOULD provide its metadata within the HTTP POST body encoded in ``application/json``.    
```

Reply:
Is the ``state`` missing?

Reply:
added as OPTIONAL according to RFC6749

2688d51182a4d12681f6e2ae56c7cbe98762dc37